portfolio/src/main/java/com/google/sps/servlets/HardCodedStringServlet.java

```diff
@@ -9,11 +11,30 @@
 /** Servlet that returns HTML that contains a hardcoded string. */
 @WebServlet("/mystring")
 public class HardCodedStringServlet extends HttpServlet {
-
+    
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> messages = new ArrayList<String>();
+    //add messages to arraylist messages
+    messages.add("I'm a golden state warriors fan!");
+    messages.add("My favorite Tv show is How to get away with murder");
+    messages.add("I play basketball");
 
-    response.setContentType("text/html;");
-    response.getWriter().println("<p>Hello World.</p>");
+    //convert the messages to json
+    String messageJson = convertToJson(messages);
+    response.setContentType("application/json;");
+    response.getWriter().println(messageJson);
+    
+  }
+/**Convert Arraylist messages into Json string */
+  private String convertToJson(ArrayList<String> messages){
```

Comment:
in java we usually pass the parameters in its more generic form. Here we can change the function to take a `List` instead of `ArrayList`. the benefit is: if we later change the implementation from ArrayList to LinkedList, this util method still works.

Reply:
Thanks, I had no idea I could do that!